Sources/MessagingPush/MessagingPush.swift

```diff
@@ -85,22 +76,13 @@ public class MessagingPush: ModuleTopLevelObject<MessagingPushInstance>, Messagi
     @available(iOS, unavailable)
     @available(iOSApplicationExtension, introduced: 13.0)
     @discardableResult
-    public static func initialize(
-        cdpApiKey: String,
-        configure configureHandler: ((inout MessagingPushConfigOptions) -> Void)? = nil
-    ) -> MessagingPushInstance {
-        if let configureHandler = configureHandler {
-            configureHandler(&moduleConfig)
-        }
-        moduleConfig.cdpApiKey = cdpApiKey
-
-        shared.initializeModuleIfNotAlready()
-
+    public static func initializeForExtension(withConfig config: MessagingPushConfigOptions) -> MessagingPushInstance {
```

Comment:
Because two `initialize` with same name and params aren't possible, I modified the name. Open for suggestions though.